### PR TITLE
Update german translation

### PR DESCRIPTION
**Pull request by dantefromhell**
To match changes in PR #23

### CODE REVIEW

**Comment by dantefromhell** [comment]
This is ready for review

@liketechnik @christoph-heiss  If you would be so kind to double-check my translation again.

**Review by jakehamilton — APPROVED** [review]
Thank you!

File: public/locales/de/translations.json [hunk 1]
```diff
@@ -3,35 +3,26 @@
 	"header.community": "Gemeinschaft",
 	"goals.title": "Unsere Ziele",
 	"goals.independent.title": "Unabhängigkeit",
-	"goals.independent": "Wir werden Nix, NixPkgs und NixOS forken und betreuen. Zusätzliche Unterstützung für Projekte wie Nix Darwin und Home Manager wird in Betracht gezogen.",
+	"goals.independent": "Wir werden eine Alternative zu NixPkgs und NixOS erstellen, die für das neue Ökosystem notwendigen Dienste, sowie eigenständige 'Continous Integration' und 'Binary Caching' Dienste bereitstellen. Zusätzliche Unterstützung für Projekte wie Nix Darwin und Home Manager wird in Betracht gezogen.",
```
Comment (liketechnik) [inline public/locales/de/translations.json]: nit: ...erstellen, *sowie* die für das neue Ökosystem notwendigen Dienste *bereitstellen*, *darunter* eigenständige 'Continous Integration' und 'Binary Caching' Dienste.

**Review by liketechnik — COMMENTED** [review]
@dantefromhell Wow, that was fast!

File: public/locales/de/translations.json (hunk 1 above)
Comment (dantefromhell) [reply public/locales/de/translations.json]: Agreed, sounds much better.

**Review by liketechnik — APPROVED** [review]

**Pull request merged**